aio_pika/robust_channel.py

```diff
@@ -42,8 +43,8 @@ def __init__(self, connection, channel_number: int = None,
             on_return_raises=on_return_raises,
         )
 
-        self._exchanges = dict()
-        self._queues = dict()
+        self._exchanges = []
```

Comment:
`defaultdict(set)` will looks better I guess.

Reply:
But why should it be a dict?